app/routes/irrigation-status.js

```diff
@@ -9,10 +9,11 @@ const currentPath = `${urlPrefix}/${viewTemplate}`
 const previousPath = `${urlPrefix}/irrigated-crops`
 const nextPath = `${urlPrefix}/irrigated-land`
 
-function createModel (errorMessage, data) {
+function createModel (errorMessage, data, hasScore) {
   return {
     backLink: previousPath,
     formActionPage: currentPath,
+    hasScore,
```

Comment:
We dont want the user to change answer here and then go straight to result page this will not be correct it will not record the correct default values for the next 3 dependent questions . so in this case we should only have continue button . Infact user shouldnt be coming back to this page after reaching the scoring page as this question is not displayed in the summary page

Reply:
that's correct - good idea
I'll challenge Ravi on this and explain to him & Shaz
Thanks :)